contexts/did-v1.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>
+DID Core v1.0 Terms
+  </title>
+  <meta content="text/html; charset=utf-8" http-equiv="Content-Type"><!--
+    === NOTA BENE ===
+    For the three scripts below, if your spec resides on dev.w3 you can check them
+    out in the same tree and use relative links so that they"ll work offline.
+   -->
+
+  <script class="remove"
+    src="https://www.w3.org/Tools/respec/respec-w3c">
+  </script><!--script src="./respec-w3c-common.js" class="remove"></script-->
+  <script class="remove" type="text/javascript">
+var respecConfig = {
+  wgPublicList: "public-did-wg",
+  group: "did",
+
+  // specification status (e.g., WD, LCWD, NOTE, etc.). If in doubt use ED.
+  specStatus: "base",
+
+  // the specification's short name, as in http://www.w3.org/TR/short-name/
+  shortName: "did-spec-registries",
```

Comment:
This cannot be right: 'did-spec-registries' is already taken, at least as short name.

Reply:
Oops, thanks. I don't think the ns doc really needs a shortname, does it

Reply:
> Oops, thanks. I don't think the ns doc really needs a shortname, does it

Probably not. But respec may (I'm not sure)